larray_editor/comparator.py

```diff
@@ -263,13 +263,20 @@ def _setup_and_check(self, widget, data, title, readonly, **kwargs):
         main_splitter.addWidget(comparatorwidget)
         main_splitter.setSizes([5, 95])
         main_splitter.setCollapsible(1, False)
+        self.widget_state_settings['{}_main_splitter'.format(self.name)] = main_splitter
```

Comment:
wouldn't it be cleaner if you didn't use self.name in the dict keys (but added it in save_widgets_state_and_geometry and restore_widgets_state_and_geometry)?